### DISCUSSION TITLE
UI blink when it comes to / root dir

### DISCUSSION THREAD

**Discussion by linepi** [nvim-tree/nvim-tree.lua#2855] [Q&A]
![Recording 2024-07-28 at 00 42 33](https://github.com/user-attachments/assets/60a572c4-9253-4159-8518-724a5292573b)

**Comment by alex-courtis** [comment]
It's likely encountering permission / special file issues at root.

Turn on [diagnostic logging](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#diagnostic-logging) for profile and watcher and you might see what's slow.
